qa-core/src/config/internal-api/TestConfiguration/rows.ts

```diff
@@ -36,4 +36,21 @@ export default class RowsApi {
     expect(response).toHaveStatusCode(200)
     return [response, json]
   }
+
+  async searchNoPagination(tableId: string, body: any): Promise<[Response, Row[]]> {
```

Comment:
Why do we have 2 methods for searching with and without pagination? Is there ever a point where we absolutely don't want pagination and need to assert on it? Seems easier just to have one `search` method

Reply:
We need to check that pagination is working, both by having multiple pages when there are more than 10 rows and only one page when there are 10 or less rows. We could have an `if` inside that method, but this is against best practices because in order to check both scenarios we would need to add logic inside the method and that would require its own tests. Each method does its own simple assertion and it will fail if the result is unexpected, as it should.